### PR TITLE
FOGL-4814 syslog API enhancements

### PR DESCRIPTION
**Pull request by ashish-jabble**
Signed-off-by: ashish-jabble <ashish@dianomic.com>

Notes:
```
Info & Above => info | warning | error | fatal
Warning & Above => warning | error | fatal
Error & Above => error | fatal
debug => all
```

### CODE REVIEW

File: python/fledge/services/core/api/support.py [hunk 1]
```diff
@@ -31,18 +32,22 @@
 __DEFAULT_OFFSET = 0
 __DEFAULT_LOG_SOURCE = 'Fledge'
 __GET_SYSLOG_CMD_TEMPLATE = "grep -a -E '({})\[' {} | head -n {} | tail -n {}"
-__GET_SYSLOG_CMD_WITH_ERROR_TEMPLATE = "grep -a -E '({})\[' {} | grep -a -E -i 'error' | head -n {} | tail -n {}"
-__GET_SYSLOG_CMD_WITH_WARNING_TEMPLATE = "grep -a -E '({})\[' {} | grep -a -E -i '(error|warning)' | head -n {} | tail -n {}"
+__GET_SYSLOG_CMD_WITH_INFO_TEMPLATE = "grep -a -E '({})\[' {} | grep -a -E -i '(info|warning|error|fatal)' | head -n {} | tail -n {}"
+__GET_SYSLOG_CMD_WITH_ERROR_TEMPLATE = "grep -a -E '({})\[' {} | grep -a -E -i '(error|fatal)' | head -n {} | tail -n {}"
+__GET_SYSLOG_CMD_WITH_WARNING_TEMPLATE = "grep -a -E '({})\[' {} | grep -a -E -i '(warning|error|fatal)' | head -n {} | tail -n {}"
+__GET_SYSLOG_CMD_WITH_DEBUG_TEMPLATE = "grep -a -E '({})\[' {} | grep -a -E -i 'debug' | head -n {} | tail -n {}"
```
Comment (praveen-garg) [inline python/fledge/services/core/api/support.py:38]: No, this is simply __GET_SYSLOG_CMD_TEMPLATE (Debug and above),  as it needs to get all **Not debug only** (debug|info|warning|error|fatal)

**Review by praveen-garg — CHANGES_REQUESTED** [review]
Also better to handle EXCEPTION along with FATAL

File: python/fledge/services/core/api/support.py (hunk 1 above)
Comment (ashish-jabble) [reply python/fledge/services/core/api/support.py:38]: There is no option Debug & Above we need to show on GUI as per Mark. This is only stays at API level for development purpose And only get debug statements.

Comment (praveen-garg) [reply python/fledge/services/core/api/support.py:38]: GUI does not matter here.  it does not make any sense to see DEBUG only grep! What value those logs do have if not with other logs?

Comment (ashish-jabble) [reply python/fledge/services/core/api/support.py]: Fixed as per your suggestions

**Review by praveen-garg — APPROVED** [review]

**Review by mshariq-nerd — APPROVED** [review]

**Pull request merged**